.github/linters/.arm-ttk.psd1

```diff
@@ -4,10 +4,12 @@
 @{
     # Test = @( )
     Skip = @(
-        'Template Should Not Contain Blanks',
+        'Template Should Not Contain Blanks'
```

Comment:
Please change "should" to "must".

Reply:
This is a predefined rule. We cannot change this or functionality of the linter would change.